Wrappers/Python/test/test_DataProcessor.py

```diff
@@ -597,6 +596,184 @@ def test_DataProcessorChaining(self):
         print("check call method of DataProcessor")
         numpy.testing.assert_array_equal(ax(chain(c)).as_array(), arr)        
 
+class TestMaskGenerator(unittest.TestCase):       
+
+    def test_MaskGenerator(self): 
+    
+        IG = ImageGeometry(voxel_num_x=10,
+                        voxel_num_y=10)
+        
+        data = IG.allocate('random')
+        
+        data.as_array()[2,3] = float('inf')
+        data.as_array()[4,5] = float('nan')
+        
+        # check special values - default
+        m = MaskGenerator()
+        m.set_input(data)
+        mask = m.process()
+        
+        mask_manual = numpy.ones((10,10), dtype=numpy.bool)
+        mask_manual[2,3] = 0
+        mask_manual[4,5] = 0
+        
+        numpy.testing.assert_allclose(mask.as_array(), mask_manual, rtol=1E-6)
```

Comment:
these are boolean arrays, should we not use `assert_array_equal`?

Reply:
fixed